src/main/java/com/linkedin/kmf/apps/SingleClusterMonitor.java

```diff
@@ -43,7 +43,6 @@
 
 public class SingleClusterMonitor implements App {
   private static final Logger LOG = LoggerFactory.getLogger(SingleClusterMonitor.class);
-
```

Comment:
Recommend adding the gradle checkstyle plugin to enforce the consistency that is desired here.  Then address all the checkstyle violations in a single PR.  After that have it fail the build on violations.

Reply:
ready in a separate PR.